Bottles!

jhheider opened the pull request:
@mxcl @jonchang: other than the, frankly, abusive misuse of `run()`, does this look sane to you?

proof that it works: https://github.com/teaxyz/homebrew-pkgs/actions/runs/5538089011

credit to GPT-4 for helping while also misleading me, per usual.

the secret sauce is that it repacks the artifacts we already generate on release to use for bottles.

mxcl commented:
@jhheider this got forgotten, can we get it working? I want to update install instructions.

jhheider committed:
provide `tea-cli` bottles


wip

wip

wip

wip

wip

wip

wip

wip

wip

wip

wip

wip

wip

wip


wip

wip

wip (better)


wip


needs more perms for GITHUB_OUTPUT


wip


wpi


ChatGPT lied to me


hopefully done


done

mxcl reviewed (approved):
beautiful work

insanely manual!

I lol’d at shelling out to `mkdir` but a minor thing. Approved.

mxcl commented:
merging as I want this today but would still love to get a @jonchang lookie

jhheider commented:
> I lol’d at shelling out to `mkdir` but a minor thing. Approved.

That's hopefully the worst sin included. It's about halfway between typescript and shellscript, and can definitely take some cleanup.
